rafs/src/metadata/inode.rs

```diff
@@ -338,7 +338,7 @@ impl InodeWrapper {
         match self {
             InodeWrapper::V5(i) => i.i_mtime,
             InodeWrapper::V6(i) => i.i_mtime,
-            InodeWrapper::Ref(_i) => unimplemented!(),
+            InodeWrapper::Ref(i) => i.get_attr().mtime,
```

Comment:
Is it necessary to implement it in other methods like `fn gid(&self)`, `fn uid(&self)`?